--- configure.old	2019-06-17 15:47:46.000000000 +0100
+++ configure	2019-07-24 07:59:39.160551000 +0100
@@ -887,6 +887,7 @@
 docdir
 oldincludedir
 includedir
+runstatedir
 localstatedir
 sharedstatedir
 sysconfdir
@@ -1009,6 +1010,7 @@
 sysconfdir='${prefix}/etc'
 sharedstatedir='${prefix}/com'
 localstatedir='${prefix}/var'
+runstatedir='${localstatedir}/run'
 includedir='${prefix}/include'
 oldincludedir='/usr/include'
 docdir='${datarootdir}/doc/${PACKAGE_TARNAME}'
@@ -1261,6 +1263,15 @@
   | -silent | --silent | --silen | --sile | --sil)
     silent=yes ;;
 
+  -runstatedir | --runstatedir | --runstatedi | --runstated \
+  | --runstate | --runstat | --runsta | --runst | --runs \
+  | --run | --ru | --r)
+    ac_prev=runstatedir ;;
+  -runstatedir=* | --runstatedir=* | --runstatedi=* | --runstated=* \
+  | --runstate=* | --runstat=* | --runsta=* | --runst=* | --runs=* \
+  | --run=* | --ru=* | --r=*)
+    runstatedir=$ac_optarg ;;
+
   -sbindir | --sbindir | --sbindi | --sbind | --sbin | --sbi | --sb)
     ac_prev=sbindir ;;
   -sbindir=* | --sbindir=* | --sbindi=* | --sbind=* | --sbin=* \
@@ -1398,7 +1409,7 @@
 for ac_var in	exec_prefix prefix bindir sbindir libexecdir datarootdir \
 		datadir sysconfdir sharedstatedir localstatedir includedir \
 		oldincludedir docdir infodir htmldir dvidir pdfdir psdir \
-		libdir localedir mandir
+		libdir localedir mandir runstatedir
 do
   eval ac_val=\$$ac_var
   # Remove trailing slashes.
@@ -1551,6 +1562,7 @@
   --sysconfdir=DIR        read-only single-machine data [PREFIX/etc]
   --sharedstatedir=DIR    modifiable architecture-independent data [PREFIX/com]
   --localstatedir=DIR     modifiable single-machine data [PREFIX/var]
+  --runstatedir=DIR       modifiable per-process data [LOCALSTATEDIR/run]
   --libdir=DIR            object code libraries [EPREFIX/lib]
   --includedir=DIR        C header files [PREFIX/include]
   --oldincludedir=DIR     C header files for non-gcc [/usr/include]
@@ -12454,7 +12466,7 @@
     We can't simply define LARGE_OFF_T to be 9223372036854775807,
     since some C++ compilers masquerading as C compilers
     incorrectly reject 9223372036854775807.  */
-#define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
+#define LARGE_OFF_T ((((off_t) 1 << 31) << 31) - 1 + (((off_t) 1 << 31) << 31))
   int off_t_is_large[(LARGE_OFF_T % 2147483629 == 721
 		       && LARGE_OFF_T % 2147483647 == 1)
 		      ? 1 : -1];
@@ -12500,7 +12512,7 @@
     We can't simply define LARGE_OFF_T to be 9223372036854775807,
     since some C++ compilers masquerading as C compilers
     incorrectly reject 9223372036854775807.  */
-#define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
+#define LARGE_OFF_T ((((off_t) 1 << 31) << 31) - 1 + (((off_t) 1 << 31) << 31))
   int off_t_is_large[(LARGE_OFF_T % 2147483629 == 721
 		       && LARGE_OFF_T % 2147483647 == 1)
 		      ? 1 : -1];
@@ -12524,7 +12536,7 @@
     We can't simply define LARGE_OFF_T to be 9223372036854775807,
     since some C++ compilers masquerading as C compilers
     incorrectly reject 9223372036854775807.  */
-#define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
+#define LARGE_OFF_T ((((off_t) 1 << 31) << 31) - 1 + (((off_t) 1 << 31) << 31))
   int off_t_is_large[(LARGE_OFF_T % 2147483629 == 721
 		       && LARGE_OFF_T % 2147483647 == 1)
 		      ? 1 : -1];
@@ -12569,7 +12581,7 @@
     We can't simply define LARGE_OFF_T to be 9223372036854775807,
     since some C++ compilers masquerading as C compilers
     incorrectly reject 9223372036854775807.  */
-#define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
+#define LARGE_OFF_T ((((off_t) 1 << 31) << 31) - 1 + (((off_t) 1 << 31) << 31))
   int off_t_is_large[(LARGE_OFF_T % 2147483629 == 721
 		       && LARGE_OFF_T % 2147483647 == 1)
 		      ? 1 : -1];
@@ -12593,7 +12605,7 @@
     We can't simply define LARGE_OFF_T to be 9223372036854775807,
     since some C++ compilers masquerading as C compilers
     incorrectly reject 9223372036854775807.  */
-#define LARGE_OFF_T (((off_t) 1 << 62) - 1 + ((off_t) 1 << 62))
+#define LARGE_OFF_T ((((off_t) 1 << 31) << 31) - 1 + (((off_t) 1 << 31) << 31))
   int off_t_is_large[(LARGE_OFF_T % 2147483629 == 721
 		       && LARGE_OFF_T % 2147483647 == 1)
 		      ? 1 : -1];
@@ -18803,11 +18815,10 @@
 
 
 
-CHANGELOG_CMD="((GIT_DIR=\$(top_srcdir)/.git git log > \$(top_srcdir)/.changelog.tmp) 2>/dev/null && \
+CHANGELOG_CMD="(GIT_DIR=\$(top_srcdir)/.git git log > \$(top_srcdir)/.changelog.tmp && \
 mv \$(top_srcdir)/.changelog.tmp \$(top_srcdir)/ChangeLog) \
-|| (rm -f \$(top_srcdir)/.changelog.tmp; test -e \$(top_srcdir)/ChangeLog || ( \
-touch \$(top_srcdir)/ChangeLog; \
-echo 'git failed to create ChangeLog: installing empty ChangeLog.' >&2))"
+|| (rm -f \$(top_srcdir)/.changelog.tmp; touch \$(top_srcdir)/ChangeLog; \
+echo 'git directory not found: installing possibly empty changelog.' >&2)"
 
 
 
@@ -18815,45 +18826,14 @@
 macros_datadir=`$PKG_CONFIG --print-errors --variable=pkgdatadir xorg-macros`
 INSTALL_CMD="(cp -f "$macros_datadir/INSTALL" \$(top_srcdir)/.INSTALL.tmp && \
 mv \$(top_srcdir)/.INSTALL.tmp \$(top_srcdir)/INSTALL) \
-|| (rm -f \$(top_srcdir)/.INSTALL.tmp; test -e \$(top_srcdir)/INSTALL || ( \
-touch \$(top_srcdir)/INSTALL; \
-echo 'failed to copy INSTALL from util-macros: installing empty INSTALL.' >&2))"
+|| (rm -f \$(top_srcdir)/.INSTALL.tmp; touch \$(top_srcdir)/INSTALL; \
+echo 'util-macros \"pkgdatadir\" from xorg-macros.pc not found: installing possibly empty INSTALL.' >&2)"
 
 
 
 
 
 
-case $host_os in
-    solaris*)
-        # Solaris 2.0 - 11.3 use SysV man page section numbers, so we
-        # check for a man page file found in later versions that use
-        # traditional section numbers instead
-        { $as_echo "$as_me:${as_lineno-$LINENO}: checking for /usr/share/man/man7/attributes.7" >&5
-$as_echo_n "checking for /usr/share/man/man7/attributes.7... " >&6; }
-if ${ac_cv_file__usr_share_man_man7_attributes_7+:} false; then :
-  $as_echo_n "(cached) " >&6
-else
-  test "$cross_compiling" = yes &&
-  as_fn_error $? "cannot check for file existence when cross compiling" "$LINENO" 5
-if test -r "/usr/share/man/man7/attributes.7"; then
-  ac_cv_file__usr_share_man_man7_attributes_7=yes
-else
-  ac_cv_file__usr_share_man_man7_attributes_7=no
-fi
-fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ac_cv_file__usr_share_man_man7_attributes_7" >&5
-$as_echo "$ac_cv_file__usr_share_man_man7_attributes_7" >&6; }
-if test "x$ac_cv_file__usr_share_man_man7_attributes_7" = xyes; then :
-  SYSV_MAN_SECTIONS=false
-else
-  SYSV_MAN_SECTIONS=true
-fi
-
-        ;;
-    *) SYSV_MAN_SECTIONS=false ;;
-esac
-
 if test x$APP_MAN_SUFFIX = x    ; then
     APP_MAN_SUFFIX=1
 fi
@@ -18869,9 +18849,9 @@
 fi
 
 if test x$FILE_MAN_SUFFIX = x    ; then
-    case $SYSV_MAN_SECTIONS in
-	true)				FILE_MAN_SUFFIX=4  ;;
-	*)				FILE_MAN_SUFFIX=5  ;;
+    case $host_os in
+	solaris*)	FILE_MAN_SUFFIX=4  ;;
+	*)		FILE_MAN_SUFFIX=5  ;;
     esac
 fi
 if test x$FILE_MAN_DIR = x    ; then
@@ -18879,9 +18859,9 @@
 fi
 
 if test x$MISC_MAN_SUFFIX = x    ; then
-    case $SYSV_MAN_SECTIONS in
-	true)				MISC_MAN_SUFFIX=5  ;;
-	*)				MISC_MAN_SUFFIX=7  ;;
+    case $host_os in
+	solaris*)	MISC_MAN_SUFFIX=5  ;;
+	*)		MISC_MAN_SUFFIX=7  ;;
     esac
 fi
 if test x$MISC_MAN_DIR = x    ; then
@@ -18889,9 +18869,9 @@
 fi
 
 if test x$DRIVER_MAN_SUFFIX = x    ; then
-    case $SYSV_MAN_SECTIONS in
-	true)				DRIVER_MAN_SUFFIX=7  ;;
-	*)				DRIVER_MAN_SUFFIX=4  ;;
+    case $host_os in
+	solaris*)	DRIVER_MAN_SUFFIX=7  ;;
+	*)		DRIVER_MAN_SUFFIX=4  ;;
     esac
 fi
 if test x$DRIVER_MAN_DIR = x    ; then
@@ -18899,9 +18879,9 @@
 fi
 
 if test x$ADMIN_MAN_SUFFIX = x    ; then
-    case $SYSV_MAN_SECTIONS in
-	true)				ADMIN_MAN_SUFFIX=1m ;;
-	*)				ADMIN_MAN_SUFFIX=8  ;;
+    case $host_os in
+	solaris*)	ADMIN_MAN_SUFFIX=1m ;;
+	*)		ADMIN_MAN_SUFFIX=8  ;;
     esac
 fi
 if test x$ADMIN_MAN_DIR = x    ; then
@@ -19918,8 +19898,8 @@
 # Checks for pkg-config packages
 
 # Always required
-X11_REQUIRES='xproto >= 7.0.17 xextproto xtrans xcb >= 1.11.1'
-X11_EXTRA_DEPS="xcb >= 1.11.1"
+X11_REQUIRES='xproto >= 7.0.17 xextproto xtrans'
+X11_EXTRA_DEPS=""
 
 
 
@@ -20374,20 +20354,6 @@
 
  fi
 
- # Other functions Xtrans may need
- for ac_func in strcasecmp strlcpy
-do :
-  as_ac_var=`$as_echo "ac_cv_func_$ac_func" | $as_tr_sh`
-ac_fn_c_check_func "$LINENO" "$ac_func" "$as_ac_var"
-if eval test \"x\$"$as_ac_var"\" = x"yes"; then :
-  cat >>confdefs.h <<_ACEOF
-#define `$as_echo "HAVE_$ac_func" | $as_tr_cpp` 1
-_ACEOF
-
-fi
-done
-
-
 
 
 # Check for dlopen
@@ -21282,6 +21248,8 @@
     if (*(data + i) != *(data3 + i))
       return 14;
   close (fd);
+  free (data);
+  free (data3);
   return 0;
 }
 _ACEOF
@@ -21791,7 +21759,7 @@
 
 
 
-ac_config_files="$ac_config_files Makefile include/Makefile man/Makefile man/xkb/Makefile src/Makefile src/util/Makefile src/xcms/Makefile src/xlibi18n/Makefile modules/Makefile modules/im/Makefile modules/im/ximcp/Makefile modules/lc/Makefile modules/lc/def/Makefile modules/lc/gen/Makefile modules/lc/Utf8/Makefile modules/om/Makefile modules/om/generic/Makefile src/xkb/Makefile nls/Makefile specs/Makefile specs/i18n/Makefile specs/i18n/compose/Makefile specs/i18n/framework/Makefile specs/i18n/localedb/Makefile specs/i18n/trans/Makefile specs/libX11/Makefile specs/XIM/Makefile specs/XKB/Makefile x11.pc x11-xcb.pc"
+ac_config_files="$ac_config_files Makefile include/Makefile man/Makefile man/xkb/Makefile src/Makefile src/util/Makefile src/xcms/Makefile src/xlibi18n/Makefile modules/Makefile modules/im/Makefile modules/im/ximcp/Makefile modules/lc/Makefile modules/lc/def/Makefile modules/lc/gen/Makefile modules/lc/Utf8/Makefile modules/om/Makefile modules/om/generic/Makefile src/xkb/Makefile nls/Makefile specs/Makefile specs/i18n/Makefile specs/i18n/compose/Makefile specs/i18n/framework/Makefile specs/i18n/localedb/Makefile specs/i18n/trans/Makefile specs/libX11/Makefile specs/XIM/Makefile specs/XKB/Makefile x11.pc"
 
 cat >confcache <<\_ACEOF
 # This file is a shell script that caches the results of configure
@@ -22894,7 +22862,6 @@
     "specs/XIM/Makefile") CONFIG_FILES="$CONFIG_FILES specs/XIM/Makefile" ;;
     "specs/XKB/Makefile") CONFIG_FILES="$CONFIG_FILES specs/XKB/Makefile" ;;
     "x11.pc") CONFIG_FILES="$CONFIG_FILES x11.pc" ;;
-    "x11-xcb.pc") CONFIG_FILES="$CONFIG_FILES x11-xcb.pc" ;;
 
   *) as_fn_error $? "invalid argument: \`$ac_config_target'" "$LINENO" 5;;
   esac
